src/app/pages/common/ix-forms/services/ix-formatter.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { StorageService } from 'app/services';
+
+@Injectable()
+export class IxFormatterService {
+  constructor(
+    private storageService: StorageService,
+  ) {}
+
+  memorySizeFormatting: (val: string | number) => string = (value: string | number) => {
```

Comment:
Not sure if `memory` is the best word for it as this method will be used for disk space and file sizes.
@dszidi Requesting help from a person with better English.

Reply:
Well, both in case of disk space and file sizes, it is still memory so I thought it was appropriate. Still, suggestions welcome.

Reply:
I agree with @undsoft that calling it "memory" could be misleading. Since you're converting bytes and bytes are actually a way to measure size of data, perhaps dataSizeFormatting or byteSizeFormatting might be more appropriate.